gffcompare.wdl

```diff
@@ -31,8 +31,8 @@ task GffCompare {
         File? noneFile # This is a wdl workaround. Please do not assign!
     }
     # This allows for the creation of output directories
-    String dirPrefix= if defined(outputDir)
-        then outputDir + "/"
+    String dirPrefix = if defined(outputDir)
+        then select_first([outputDir]) + "/"
```

Comment:
Out of curiosity, why is the use of select_first better here?

Reply:
outputDir is optional, so the if-then-else would have returned either an optional or a non-optional. This doesn't matter practically, but miniwdl check didn't like it.